### PR TITLE
tree-building: rm stub with full solution

### PR DESCRIPTION
**Pull request by ferhatelmas**
not full but it's not a stub

### CODE REVIEW

File: exercises/tree-building/tree_building.go [hunk 1]
```diff
@@ -1,107 +0,0 @@
...
-	} else {
-		for i := 0; i < len(n.Children); i++ {
-			err = chk(n.Children[i], m)
-			if err != nil {
-				return
-			}
-		}
-		return
-	}
-}
```
Comment (leenipper) [inline exercises/tree-building/tree_building.go]: I wondered why this stub was there originally, so I checked the README.md, and found this sentence:

> Your job will be to refactor a working but slow and ugly piece of code that implements the tree building logic for highly abstracted records.

That sentence originates in the description.md from problem-specifications.
So, should we simply keep it, or delete it and update the README.md?  The latter would make the Go exercise take a different slant than other language tracks whose focus is a refactor.

Comment (ferhatelmas) [reply exercises/tree-building/tree_building.go]: Yes, let's update readme too.

**Comment by tleen** [comment]
My comment got a little lost as a pending review. Unsure if it was seen. But I am under the impression that the bad code is part of the exercise and refactoring it is the purpose.

**Comment by leenipper** [comment]
Which way are we going with this exercise... shall we keep the stub since it is a re-factor exercise, or proceed to update the readme to remove the refactor focus and remove the stub?  If it's the former, we can close this PR without a merge.

**Comment by tleen** [comment]
I'm of the opinion if we have to completely override [the original README](https://github.com/exercism/problem-specifications/blob/master/exercises/tree-building/description.md) from `problem-specifications` then it should probably be its own independent exercise. So I would have to go with no-merge.

**Comment by leenipper** [comment]
> no-merge.

I'm thinking the same way.

**Comment by ferhatelmas** [comment]
OK :dancer:

**Pull request closed without merging**